cli.py

```diff
@@ -91,7 +91,7 @@ def __init__(self, work_dir, vmlinux, makefile, sym_vers):
         self.mod_files = self.config['mod_files']
         self.mod_srcs = [f for f in self.mod_files if f.endswith('.c')]
         self.mod_hdrs = [f for f in self.mod_files if f.endswith('.h')]
-        self.mod_objs = [f[:-2]+'.o' for f in self.mod_srcs] + ['kernel/sched/fake.o']
+        self.mod_objs = [f+'.extract' for f in self.mod_files]
```

Comment:
It seems `fake.c` is not needed anymore. We can delete this file in this patch.

Reply:
the last patch will remove fake.c.